### PR TITLE
Update readme md

### PR DESCRIPTION
**Pull request by linaya2004**
updated the README.MD filw

### CODE REVIEW

File: custom_static/css/hardhat.css
Comment (s-bonnor) [inline custom_static/css/hardhat.css:1]: based on the pull request description this file should not be included

File: custom_static/js/load_more.js
Comment (s-bonnor) [inline custom_static/js/load_more.js:1]: based on the pull request description this file should not be included

File: home/templates/pages/search-results.html
Comment (s-bonnor) [inline home/templates/pages/search-results.html:1]: based on the pull request description this file should not be included

**Review by s-bonnor — CHANGES_REQUESTED** [review]
new README looks great

File: custom_static/js/load_more.js
Comment (linaya2004) [reply custom_static/js/load_more.js:1]: I have deleted those files as requested

**Comment by linaya2004** [comment]
Hi sam, i made the changes as requested.

**Pull request closed without merging**